holoviews/plotting/bokeh/element.py

```diff
@@ -2407,10 +2410,14 @@ def _draw_scalebar(self, plot):
         `scalebar_opts`, and `scalebar_unit` for more information.
 
         Requires Bokeh 3.4
+
+        For scalebar on a subcoordinate_y plot Bokeh 3.6 is needed.
         """
 
         if not bokeh34:
             raise RuntimeError("Scalebar requires Bokeh >= 3.4.0")
+        elif not bokeh36 and self._subcoord_overlaid:
+            raise RuntimeError("Scalebar with subcoordinate_y requires Bokeh >= 3.6.0")
```

Comment:
Maybe just warn here not to break backward compatibility.

Reply:
Yes, warning is fine.